src/managing-workflow/upgrading-workflow.md

```diff
@@ -9,3 +9,24 @@ $ helmc fetch deis/workflow-beta4
 $ helmc generate -x manifests workflow-beta4
 $ helmc install workflow-beta4
 ```
+
+## Keeping Essential Pieces
+
+!!! note
+    "Keeper" upgrade behavior requires Helm Classic 0.8.0 or newer and the workflow-rc1
+    or workflow-dev Chart.
+
+Helm Classic recognizes when a Chart manifest is marked as a "keeper" and will leave it alone
+during `helmc uninstall`.
+
+The Workflow Chart marks the "deis" Kubernetes `Namespace` and the `Service` for the registry
+and for the router as keepers. This keeps the external `LoadBalancer` intact so that routing
+and DNS are preserved while reinstalling Workflow.
+
+To remove Workflow completely from a Kubernetes cluster, run the following:
+
+```
+$ helmc uninstall -n deis -y workflow-dev
+$ kubectl delete ns deis  # Be sure you want to delete the
+                          # Namespace and all its contents!
+```
```

Comment:
I think the second paragraph describes why Workflow uses "keeper" (yes, for upgrade), but let me know how I can clarify. The general `helm-keep` mechanism is described at <http://helm-classic.readthedocs.io/en/latest/awesome/#keeper-manifests>. Would it help to link to that doc from here?